<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>431</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="my_grid"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_newgame">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 255, 0);</string>
         </property>
         <property name="text">
          <string>NewGame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_check">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string>CheckGame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_darkmode">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(206, 206, 206);</string>
         </property>
         <property name="text">
          <string>DarkMode</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionItem1">
   <property name="text">
    <string>Item1</string>
   </property>
  </action>
  <action name="actionItem2">
   <property name="text">
    <string>Item2</string>
   </property>
  </action>
  <action name="actionItem1_2">
   <property name="text">
    <string>Item1</string>
   </property>
  </action>
  <action name="actionItem1_3">
   <property name="text">
    <string>Item1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
